tests/Test.MSAL.NET.Unit.PublicApi/ReadMe.txt

```diff
@@ -1,7 +0,0 @@
-﻿Unit tests for the public API only
-----------------------------------
-
-It is important this project is *not* able to see the internals of the product assemblies,
-which is why these tests are in a separate project (the product internals are visible to
-"Test.MSAL.Net.Unit").
```

Comment:
What was the reason for merging the two test projects? This one was deliberately separated to make sure it could only be testing the public API.